menu.cpp

```diff
@@ -2176,7 +2170,7 @@ void HandleUI(void)
 		// use left/right to adjust video position
 		if (left || right) {
 			if ((menusub == 2) || (menusub == 3)) {
-				if (left && (tos_get_video_adjust(menusub - 2) > -100))
```

Comment:
`char` is unsigned on ARM so this comparison is always true unless the result of `tos_get_video_adjust` is cast to `signed char`.

Reply:
right, but all MENU_MIST_* cases are used currently as it's related to MiST's Atari ST core which is not ported(yet). MiSTer's Atari ST core doesn't use these menus. So basically it's dead code which will be removed when ST core will become open source. The code is still here because i'm not sure if ST core will be open source or not.

Reply:
Oh, I see.